chat-ui/src/services/conversations.ts

```diff
@@ -26,6 +26,15 @@ export function formatReferences(references: References): string {
   return [heading, ...listOfLinks].join("\n\n");
 }
 
+export const CUSTOM_REQUEST_ORIGIN_HEADER = "x-request-origin";
```

Comment:
```suggestion
export const CUSTOM_REQUEST_ORIGIN_HEADER = "X-Request-Origin";
```

for consistency w above

Reply:
This is a good example of something that would make sense in `chat-core` but not `rag-core`